src/webview/components/Atoms/LastSync/LastSync.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import './LastSync.css';
 
-const LastSync: React.FC = () => {
```

Comment:
Recommend not using React.FC anymore I was looking into it and its fully redundant in modern React now 😭